lib/non_duplicate_sms_ids.rb

```diff
@@ -1,52 +1,23 @@
 class NonDuplicateSmsIds
```

Comment:
what do you think about renaming this now? the original name was a pretty average anyway.